CHANGELOG.md

```diff
@@ -0,0 +1,12 @@
+# Version 0.2.0
+## Features
```

Comment:
Can you please reformat this file to match this set of rules: https://dlaa.me/markdownlint/